app/src/main/java/com/josycom/mayorjay/flowoverstack/ui/MainActivity.java

```diff
@@ -18,7 +19,25 @@
 import android.view.animation.Animation;
 import android.view.animation.AnimationUtils;
 
-public class MainActivity extends AppCompatActivity {
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjection;
+import dagger.android.AndroidInjector;
+import dagger.android.DispatchingAndroidInjector;
+import dagger.android.HasAndroidInjector;
+
+public class MainActivity extends AppCompatActivity implements HasAndroidInjector {
+
+    /*
+     * Step 1: Rather than injecting the ViewModelFactory
+     * in the activity, we are going to implement the
+     * HasActivityInjector and inject the ViewModelFactory
+     * into our MovieListFragment
```

Comment:
Please check this comment, there is nothing like MovieListFragment in this project